doc/google-assistant.md

```diff
@@ -0,0 +1,62 @@
+# Google Assistant
+
+You can use Google Assistant to access Thingpedia devices!
+
+First, follow the instructions [here](https://developers.google.com/assistant/actions/actions-sdk) to create an Action package for Google Assistant.
+
+Specifically, the Action package JSON file should be replaced with the following, taking care to replace `<YourActionName>` with your Action name.
+
+```
+{
+  "actions": [
+    {
+      "description": "Default Welcome Intent",
+      "name": "MAIN",
+      "fulfillment": {
+        "conversationName": "<YourActionName>"
+      },
+      "intent": {
+        "name": "actions.intent.MAIN",
+        "trigger": {
+          "queryPatterns": [
+            "talk to <YourActionName>"
+          ]
+        }
+      }
+    },
+    {
+      "name": "TEXT",
+      "intent": {
+        "name": "actions.intent.TEXT",
+        "trigger": {
+          "queryPatterns": [
+            "bye",
+            "goodbye"
```

Comment:
Can we have some more documentation about this JSON? For example, what are these query patterns?

Reply:
Added some documentation on Action package file